bentoml/frameworks/keras.py

```diff
@@ -201,22 +200,23 @@ def load(self, path):
                 open(self._custom_objects_path(path), 'rb')
             )
 
-        with self.graph.as_default():
-            with self.sess.as_default():
-                # load keras model via json and weights if requested
-                if self._store_as_json_and_weights:
+        # load keras model via json and weights if requested
+        if self._store_as_json_and_weights:
```

Comment:
Could you add a test case for `_store_as_json_and_weights = True` option in the integration tests, in a follow-up PR?

Reply:
Already included